src/main/java/org/apereo/portlet/soffit/connector/SoffitConnectorController.java

```diff
@@ -118,6 +118,9 @@ private SoffitRequest buildSoffitRequest(final RenderRequest req, final RenderRe
         // Namespace
         rslt.setNamespace(res.getNamespace());
 
+        // User
+        rslt.getUserDetails().setUsername(req.getUserPrincipal().getName());
```

Comment:
Just a heads up -- this whole data model about to be replaced by this:  https://github.com/drewwills/Soffit/wiki/Soffit-Connector-Data-Model